bedrock/mozorg/templates/mozorg/about/leadership/includes/boards.html

```diff
@@ -345,7 +349,8 @@ <h2 class="group-title" id="board-foundation">Mozilla Foundation</h2>
             'width': '160',
             'height': '160',
             'class': 'photo',
-            'itemprop': 'image'
+            'itemprop': 'image',
+            'alt': 'Mitchell Baker, Chair'
           }
         ) }}
         <figcaption><h4 class="fn" itemprop="name">Mitchell Baker, Chair</h4></figcaption>
```

Comment:
_Q, maybe for others to also chime in on best practice:_ This could make a bit more redundancy when read by assistive tech, as it already matches what's in the figcaption and h4 heading for that content.

Would a new, unique text as "Photo of Mitchell Baker" be a better alternative?

Reply:
I found [this university page ](https://sc.edu/about/offices_and_divisions/digital-accessibility/toolbox/best_practices/alternative_text/headshot-alt-text/index.php)talking about headshot alt-texts and they mentioned we should include that it is a headshot image. So maybe something like `Headshot of Mitchell Baker, [role at mozilla]` would be a better way of writing alt-text for these images?

Reply:
I was actually thinking about clearly saying it's a "headshot" right away;) It basically sets the mood for what you can't see in the image, and is completely in line with that a11y recommendation: 

> "Just by saying that the image is a headshot, you convey several things about the image and person in it: They are a professional, likely wearing business casual and almost certainly smiling."

(Reason I recommended just "photo" in the end is the fact these pages are not localised and English is used for all visitors, globally. "Photo" sounds more universally understandable (think "Simple English") than what a "headshot" is for non-English speakers. Not sure if there's a better way to describe headshots somewhat jargon-free, like a "Profile photo of …"?)

I just think the actual role or position is not a good image description, b/c that's not something you can "see" in the photo. There might be more occurrences of the same photo (same person, different role), and its content doesn't change based on the role, but the image has different alt in each case.

Reply:
I think I read about avoiding using "photo of ... " or "image of ... " since it is the alt for images, so it's implied and not necessary (If I find the link, I will share it in the edit). I like the reasoning behind "headshot of ... " to convey the professional setting of the image. I think alt doesn't only describe the image itself, it also convey the impression/feel of a page, like what's demoed in Example 4 on this page: https://www.w3.org/WAI/tutorials/images/informative/. So "headshot" works better than just "photo" or "profile"